project/Merging.scala

```diff
@@ -16,6 +16,7 @@ object Merging {
     case PathList("META-INF", "kotlin-stdlib-common.kotlin_module") => MergeStrategy.first
     case PathList("META-INF", "okio.kotlin_module") => MergeStrategy.first
     case PathList("META-INF", "versions", "9", "OSGI-INF", "MANIFEST.MF") => MergeStrategy.first
+    case PathList("META-INF", "license", "LICENSE.mvn-wrapper.txt") => MergeStrategy.first
```

Comment:
some dependencies changed, such that we now have an assembly merge conflict on `META-INF/license/LICENSE.mvn-wrapper.txt`; added a case for this file.